cg/meta/report/api.py

```diff
@@ -282,3 +289,13 @@ def _incorporate_lims_data(self, report_data: dict):
             sample['application'] = ReportAPI._present_string(lims_sample.get('application'))
             sample['application_version'] = lims_sample.get('application_version')
             sample['received'] = ReportAPI._present_date(lims_sample.get('received'))
+
+    def _get_genes_from_scout(self, panels) -> list:
```

Comment:
We seem to be using type hints consistently throughout the code. You can add a typehint for `panels` as well